pandas/tests/frame/methods/test_copy.py

```diff
@@ -0,0 +1,21 @@
+from pandas import DataFrame
+import pandas._testing as tm
+
+
+class TestCopy:
+    def test_cache_on_copy(self):
```

Comment:
put this in pandas/tests/frame/methods/test_api with the rest of the copy tests.

Reply:
I only see other copy tests in pandas/tests/frame/test_api.py. I'll assume you mean that one.

Reply:
yes